hamocc/carchm.F90

```diff
@@ -137,9 +135,11 @@ SUBROUTINE CARCHM(kpie,kpje,kpke,kbnd,                                  &
       REAL,    intent(in) :: pfu10(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)
       REAL,    intent(in) :: ptho(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd,kpke)
       REAL,    intent(in) :: psao(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd,kpke)
+      REAL,    intent(in) :: pflxdms(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)
+      REAL,    intent(in) :: pflxbromo(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)
 
       ! Local variables
-      INTEGER :: i,j,k,l,js
+      INTEGER :: i,j,k,l,js,ks
```

Comment:
ks isn't a local variable, it needs to be imported from `mo_param1_bgc`

Reply:
Thanks for catching this. I'm concerned that the testing we have is not really covering all the use cases that we need to have identified these problems. It would be good to add a few tests that would create baselines and would have caught these problems as part of the regression testing. A quick chat to out outline what we want to do moving forwards would be very helpful. I'm happy to implement that.